tests/models/test_hparams.py

```diff
@@ -642,3 +649,30 @@ def test_model_with_fsspec_as_parameter(tmpdir):
     )
     trainer.fit(model)
     trainer.test()
+
+
+@pytest.mark.skipif(not HYDRA_AVAILABLE, reason="Hydra is not available")
+def test_model_save_hyper_parameters_interpolation_with_hydra(tmpdir):
+
```

Comment:
Mind adding a docstring mentioning the existence and path of the configuration files? this way they are linked and can be found via grep

Reply:
Done !

Reply:
Done !